unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/Tests/FacialFeatureControllerShould.cs

```diff
@@ -0,0 +1,102 @@
+using System.Collections;
+using DCL;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using WaitUntil = DCL.WaitUntil;
+
+namespace AvatarShape_Tests
+{
+    public class FacialFeatureControllerShould : TestsBase
+    {
+        private const string EYES_ID = "dcl://base-avatars/f_eyes_01";
+        private WearableDictionary catalog;
+
+        //TODO: Remove it once we have a mocking library
+        private class Mock_BodyShapeController : IBodyShapeController
+        {
+            public string bodyShapeId => WearableLiterals.BodyShapes.FEMALE;
+            public void SetupEyes(Material material, Texture texture, Texture mask, Color color) { }
+            public void SetupEyebrows(Material material, Texture texture, Color color) { }
+            public void SetupMouth(Material material, Texture texture, Color color) { }
+        }
```

Comment:
Couldn't we use our new mocking library NSubstitute here?

Reply:
Yeah, this PR is older than the one where NSubstitue was integrated. Gonna put the branch up to date and change it, nice catch

Reply:
Beautiful! Thanks! :)